### ISSUE TITLE
You can break Minefield Sign while Techies building it.

### ISSUE THREAD

**Issue by G3ST3RR** [ValveSoftware/Dota2-Gameplay#8440]
### Ability name

Minefield Sign

### Description

You can break Minefield Sign while Techies building it. [Video on Reddit](https://www.reddit.com/r/DotA2/comments/12w16i9/you_can_break_minefield_sign_during_its_cast_and/)

### Example Match ID (and possibly Timestamp)

7120964434 - 47:21

**Comment by lainv0hndyrec** [comment]
yeah this is annoying, it shows the effects and animation when building it and it being activated but no damage.
